source/lambda/kendraIndexCreator/lambda_function.py

```diff
@@ -0,0 +1,51 @@
+import boto3
+import os
+
+def on_create(event, context):
+    kendra_client = boto3.client('kendra')
+    kendraIndexId = None
+    try:
+        print("Creating Kendra Index")
+        roleArn = os.environ['KENDRA_ROLE_ARN']
+        kmsKeyId = os.environ['KMS_KEY_ID']
+        response = kendra_client.create_index(
+            Name='DUSKendraIndex',
+            Edition='DEVELOPER_EDITION',
+            RoleArn= roleArn,
+            ServerSideEncryptionConfiguration= {
+                'KmsKeyId': kmsKeyId
+            },
+            Description='Indexes Covid pdfs',
+            ClientToken=os.environ['KENDRA_INDEX_CLIENT_TOKEN'],
+        )
+        kendraIndexId = response['Id']
+    except Exception as e:
+        print(e)
```

Comment:
You would need to throw an error here or else the code would falsely consider it as a successful deployment